src/app/bulletin-test-page/page.tsx

```diff
@@ -0,0 +1,29 @@
+'use client';
+import React from 'react';
+
+import BulletinCard from '@/components/ui/BulletinCard';
+import { useBulletins } from '@/lib/queries/bulletinQueries';
+const Test: React.FC = () => {
+  const { data: bulletins, isLoading, error } = useBulletins();
+
+  if (isLoading) return <p>Loading bulletins...</p>;
+  if (error) return <p>Error fetching bulletins: {error.message}</p>;
```

Comment:
Don't display the `error.message` or `Error fetching bulletins` in the front end since this would not make sense to the user. Make it more descriptive like "An error occurred"